client/src/pages/Apparels.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { readApparelListing } from '../data';
+import { Link } from 'react-router-dom';
+import Header from '../components/header';
+
+export default function Apparels() {
+  const [apparels, setApparels] = useState([]);
+  const [error, setError] = useState();
+  const [isLoading, setIsLoading] = useState();
+
+  useEffect(() => {
+    async function load() {
+      setIsLoading(true);
+      try {
+        const shoeData = await readApparelListing();
+        setApparels(shoeData);
+      } catch (error) {
+        setError(error);
+      } finally {
+        setIsLoading(false);
+      }
+    }
+    if (isLoading === undefined) load();
+  }, [isLoading]);
+
+  if (isLoading) return <div>Loading...</div>;
+  if (error) return <div>Error loading list: {error.message}</div>;
+  if (!apparels) return null;
+
+  return (
+    <>
+      <Header />
```

Comment:
You can put Header here in all your page components, or you can use the Outlet approach that's in react-routing. This works, so it's not a problem.